src/core/crypto/multiKey.ts

```diff
@@ -1,12 +1,39 @@
+import { AuthenticationKey } from "../authenticationKey";
 import { Hex } from "../hex";
-import { HexInput } from "../../types";
+import { HexInput, SigningScheme as AuthenticationKeyScheme } from "../../types";
 import { Deserializer } from "../../bcs/deserializer";
-import { Serializer } from "../../bcs/serializer";
-import { AnyPublicKey } from "./anyPublicKey";
-import { AnySignature } from "./anySignature";
-import { PublicKey } from "./asymmetricCrypto";
+import { Serializable, Serializer } from "../../bcs/serializer";
+import { Ed25519PublicKey, Ed25519Signature } from "./ed25519";
+import { Secp256k1PublicKey, Secp256k1Signature } from "./secp256k1";
+import { AnyPublicKey, AnySignature } from "./singleKey";
+
+type PublicKeyInput = Ed25519PublicKey | Secp256k1PublicKey | AnyPublicKey;
+type SignatureInput = Ed25519Signature | Secp256k1Signature | AnySignature;
+
+/* eslint-disable no-bitwise */
+
+function bitCount(byte: number) {
+  let n = byte;
+  n -= (n >> 1) & 0x55555555;
+  n = (n & 0x33333333) + ((n >> 2) & 0x33333333);
+  return (((n + (n >> 4)) & 0xf0f0f0f) * 0x1010101) >> 24;
+}
+
+/* eslint-enable no-bitwise */
```

Comment:
remove?

Reply:
That's needed.
There's a global rule for not using bitwise operations, but we need to use them in that function.
Alternatively, we can just globally remove the rule?